hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/RingbufferContainer.java

```diff
@@ -464,8 +520,11 @@ public void writeData(ObjectDataOutput out) throws IOException {
     @Override
     @SuppressWarnings("unchecked")
     public void readData(ObjectDataInput in) throws IOException {
-        name = in.readUTF();
-        emptyRingWaitNotifyKey = new RingbufferWaitNotifyKey(name, EMPTY);
+        assert !in.getVersion().isUnknown();
```

Comment:
Seems this is the case : https://github.com/hazelcast/hazelcast-enterprise/blob/488e4ea80503a19d048a44b0271a61b7acb3a6d1/hazelcast-enterprise/src/main/java/com/hazelcast/internal/serialization/impl/EnterpriseDataSerializableSerializer.java#L146.
I'll adapt the `readData` and `writeData` then.